src/features/common/types/types.ts

```diff
@@ -1,31 +1,19 @@
-// export type DataResponseType = {
-//     page: number
-//     per_page: number
-//     total: number
-//     total_pages: number
-//     data:
-// }
-//
-// export type UserType = {
-//     id: number
-// }
+export type ResponseDataType = {
```

Comment:
Let's remove the `src/features/common/types` folder, we can make use of `Ambient Type Declarations` feature to use our global types without explicit imports. We just need to create a `src/types.d.ts` file, add all the type declared in the current module there and remove `export` operators. Let me know if you need more details about it

Reply:
@Acousticdesk It's nice refactor. I will use it in the future!